content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
 
-- To manually add an annotation, left-click anywhere in your graph, and click **Add annotation**.
-- In the **Description** box, type "Migrated user database".
-- In the **Tags** box, type "migration".
+Adding annotations to your dashboards is a great way to communicate important events to the rest of your team.
 
-Grafana also lets you annotate a time interval, through _region annotations_.
+When things go bad, it often helps if you understand the context in which the system failed. Time of last deploy, system changes, or database migration can offer insight into what might have caused an outage. Annotations allow you to add custom events to your graphs.
 
-- While pressing Ctrl (or Cmd on macOS), click and drag across the graph using your left mouse button.
-- In the **Description** box, type "Performed load tests".
-- In the **Tags** box, type "testing".
+In the next part of the tutorial, we will simulate some common use cases that someone would add annotations for.
 
-Adding annotations to your dashboards is a great way to communicate important events to the rest of your team.
+- To manually add an annotation, click anywhere in your graph, then click **Add annotation**.
+- In **Description**, enter **Migrated user database**.
+- In **Tags**, enter **migration**.
+- Click **Save**.
+  Grafana adds your annotation to the graph. Hover your mouse over the base of the annotation to read the text.
+
+((Marcus - Since we are having them tag the annotations, you might want to demonstrate how to use them, or at least explain what they do.))
+Grafana also lets you annotate a time interval, with _region annotations_.
+
+Add a region annotation.
 
-Manually annotating your dashboard is fine for those one-off events. For regularly occurring events, such as deploying a new release, Grafana supports querying annotations from one of your data sources:
+- Press Ctrl (or Cmd on macOS), then click and drag across the graph to select an area.
+- In **Description**, enter **Performed load tests**.
+- In **Tags**, enter **testing**.
+((In case you wondered, I'm changing "type" to "enter" because there are multiple ways to input the value. If we specify type, then they might think they cannot copy and paste. In reality, we don't care how they get the value there, so we say "enter."))
 
-- Select **Dashboard settings** from the top of the dashboard view.
-- Click **Annotations**, then **New Annotation Query**.
-- In the **Name** box, type "Errors".
-- Select "Loki" from the **Data source** drop down.
-- In the **Query** box, type a LogQL query:
+Manually annotating your dashboard is fine for those single events. For regularly occurring events, such as deploying a new release, Grafana supports querying annotations from one of your data sources. Let's create an annotation using the Loki data source we added earlier.
+((One hallmark of tutorials is friendly language like this. We hold their hand and walk them through what we are doing and why.))
```

Comment:
Interesting. I've been refraining from using "Let's" as it's referring to "us", and not the user, "you". I'd love to chat more on this.

Reply:
Honestly, you have a lot of leeway in regards to style as long as you are consistent. Just seemed like Grafana style skews towards being friendly and casual, and I felt like there needed to be more of a transition in here.